### PR TITLE
Added localizable

### PR DESCRIPTION
**Pull request by IgnacioAbadie**
## Summary ##

https://trello.com/c/dSzyflIp/12-n-421-localizacion

Localizable

### spanish

Captura de Pantalla 2020-01-23 a la(s) 13 06 08

### english
Captura de Pantalla 2020-01-23 a la(s) 13 06 31

### CODE REVIEW

**Review by gmazzei — APPROVED** [review]
Great! 🏆

File: WBooks/Localizable/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* 
+  Localizable.strings
+  WBooks
+
+  Created by Nacho 2 on 23/01/2020.
+  Copyright © 2020 Wolox. All rights reserved.
+*/
+
+/* MAIN VIEW */
```
Comment (carcos-wolox) [inline WBooks/Localizable/en.lproj/Localizable.strings]: I prefer to use `// MARK: - Main View` so you can access them easily later

**Review by carcos-wolox — APPROVED** [review]
Nice! remember to add all your strings here :D

**Comment by IgnacioAbadie** [comment]
Added for Login
image

**Pull request merged**